src/lib/controls/onvif/manager.rs

```diff
@@ -283,7 +283,7 @@ impl Manager {
 
         let stream_information = streams_information
             .iter()
-            .find(|&stream_information| &stream_information.stream_uri.to_string() == stream_uri)
+            .find(|&stream_information| stream_information.stream_uri.to_string() == *stream_uri)
```

Comment:
I believe that the commit title does not represent the code change

Reply:
Thanks! I removed the commit since it wasn't necessary.